package/package.json

```diff
@@ -29,10 +29,37 @@
 	},
 	"type": "module",
 	"peerDependencies": {
-		"astro": "^4.0.0"
+		"@vitejs/plugin-react": "^4.2.1",
+		"astro": "^4.4.1",
+		"preact": "^10.19.4",
+		"react": "^18.2.0",
+		"solid-js": "^1.8.15",
+		"svelte": "^4.2.11",
+		"vue": "^3.4.19"
+	},
+	"peerDependenciesMeta": {
+		"@vitejs/plugin-react": {
+			"optional": true
+		},
+		"preact": {
+			"optional": true
+		},
+		"react": {
+			"optional": true
+		},
+		"solid-js": {
+			"optional": true
+		},
+		"svelte": {
+			"optional": true
+		},
+		"vue": {
+			"optional": true
+		}
 	},
 	"devDependencies": {
 		"@types/node": "^20.11.0",
+		"astro": "^4.4.1",
```

Comment:
is this needed?

Reply:
I guess? So we can access the astro types in the utils for the hook params and stuff?